### ISSUE TITLE
Drell-Yan Estimation

### ISSUE THREAD

**Issue by demarley** [cms-tamu/HHWWbb_analysis#4]
Setup framework for interacting with DY samples to perform proper DY estimation (using BDT).

**Comment by lpernie** [comment]
Yes, I need good root file that I can load, and then I can start!

**Comment by tahuang1991** [comment]
Done.  A new version is copied to cmstca: /data/taohuang/HHNtuple_20180418_DYestimation
 The files  for each process are compressed into one so in fact there are two copies of Ntuples here: one under each subdirectory and one just under this directory with production name + "_Friend.root"

**Comment by demarley** [comment]
Thanks @tahuang1991! Now that there are some files @lpernie can update the framework to do thes estimation.

**Comment by lpernie** [comment]
I will stat with  simple script that covert those file into CSV files.
Then I will try to figure out how to convert the BDTk-based algorithm from Tao in a similar script that use sklearn or similar. Feel free to suggest!
